middlewares/Locale.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace RainLab\Translate\Middlewares;
+
+use RainLab\Translate\Classes\Translator;
+use Closure;
+
+class Locale
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \Closure  $next
+     * @return mixed
+     */
+    public function handle($request, Closure $next)
+    {
+        $translator = Translator::instance();
+        $translator->isConfigured();
+    
+        if( !$translator->loadLocaleFromRequest() ) {
```

Comment:
Could you please add some comments (either directly in the code or just here on Github) explaining exactly what is happening here?
Is this just copied over from the App::before()? Why has a lot of the other logic there been removed?